front/gatsby/gatsby-ssr.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018-2020 @paritytech/Nomidot authors & contributors
```

Comment:
I would do like that guy, put this in a file and import it from gatsby-{ssr,browser}